test/facade_converter_test.ts

```diff
@@ -107,6 +123,13 @@ ByteBuffer buff;`);
 
 var y = x is Promise;`);
     });
+
+    it('works with a different typings directory', () => {
+      expectWithTypingsRootTypes('var y: Promise = Promise.resolve(1);')
```

Comment:
The `getSources` files are compiled along the file, so they're ambient.

Do you want to falsify the test? Is this what you want? I could verify that `import {Promise} from 'another/path/to/es6-promise/es6-promise';` would not cause it to be renamed.